science/propka/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+PortGroup               github 1.0
+
+name                    propka
+revision                0
+
+supported_archs         noarch
+platforms               {darwin any}
+license                 LGPL-2.1
+maintainers             {gmail.com:howarth.at.macports @jwhowarth} openmaintainer
+
+description             Heuristic pKa calculations with ligands
+long_description        {*}${description}
+
+# The PyPi copy of propka 2.0.1 has a broken testsuite so
+# use the Jensen Group's copy from GitHub.
+
+github.setup            jensengroup propka 3.5.1 v
+github.tarball_from     releases
+distfiles               propka-v${version}.tar.gz:propka
+
+checksums               rmd160  e8b551656c19ad3777f42095ea564935ef042cfc \
+                        sha256  e7357db9dfbeeee5ea67791e09710557980555808371253c674ae4f13b9c22f0 \
+                        size    376176
+
+python.versions         312
```

Comment:
this has no effect for ports that are not named `py-<portname>` as I explained in the previous PR. Instead, you should use `python.default_version 312`